src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,49 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        // create
+        DriverService driverService
+                = (DriverService) injector.getInstance(DriverService.class);
+        Driver driverDima = new Driver(1L, "Dima", "12345");
+        Driver driverOleg = new Driver(1L, "Oleg", "56789");
+        driverService.create(driverDima);
+        driverService.create(driverOleg);
+        ManufacturerService manufacturerService
+                = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        Manufacturer bmw = new Manufacturer(1L, "BMW", "Germany");
+        Manufacturer vw = new Manufacturer(1L, "VW", "Germany");
```

Comment:
when you add a new row with the model you can use null instead of index
```suggestion
        Driver driverDima = new Driver(null, "Dima", "12345");
        Driver driverOleg = new Driver(null, "Oleg", "56789");
        driverService.create(driverDima);
        driverService.create(driverOleg);
        ManufacturerService manufacturerService
                = (ManufacturerService) injector.getInstance(ManufacturerService.class);
        Manufacturer bmw = new Manufacturer(null, "BMW", "Germany");
        Manufacturer vw = new Manufacturer(null, "VW", "Germany");
```